onnxruntime/core/framework/session_state.h

```diff
@@ -503,7 +503,11 @@ class SessionState {
   PrepackedWeightsContainer* const prepacked_weights_container_{};
 
 #if !defined(ORT_MINIMAL_BUILD)
+#ifndef DISABLE_ABSEIL
   InlinedHashMap<InlinedVector<int>, InlinedHashSet<NodeIndex>> to_be_executed_nodes_;
+#else
+  std::map<InlinedVector<int>, InlinedHashSet<NodeIndex>> to_be_executed_nodes_;
```

Comment:
why did this change from InlinedHashMap to std::map?

Reply:
std::unordered_map does not accept std::vector as a key. I reverted it to std::map, which was the original type before refactoring.